### PR TITLE
[Catalog] Add MDCActionExtension BUILD file

### LINKED ISSUE #6293: [Catalog] Add ActionExtension targets to bazel
This was filed as an internal issue. If you are a Googler, please visit [b/122550549](http://b/122550549) for more details.

---
#### Internal data
- Associated internal bug: [b/122550549](http://b/122550549)

### PR DESCRIPTION
**Pull request by andrewoverton**
Closes #6293. I guess haha. Not really sure if this is right...

### CODE REVIEW

File: catalog/MDCActionExtension/BUILD [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# limitations under the License.
+
+load(
+    "//:material_components_ios.bzl",
+    "mdc_objc_library",
+)
+
+licenses(["notice"])  # Apache 2.0
+
+mdc_objc_library(
```
Comment (romoore) [inline catalog/MDCActionExtension/BUILD]: Can we put this in the catalog BUILD file?

Comment (andrewoverton) [reply catalog/MDCActionExtension/BUILD]: Done.

**Review by romoore — APPROVED** [review]

File: catalog/BUILD [hunk 2]
```diff
@@ -127,3 +127,9 @@ swift_library(
         "@catalog_by_convention//:CatalogByConvention",
     ],
 )
+
+mdc_objc_library(
+    name = "MDCActionExtension",
+    srcs = native.glob(["MDCActionExtension/*.m"]),
+    hdrs = native.glob(["MDCActionExtension/*.h"]),
```
Comment (romoore) [inline catalog/BUILD:134]: Shouldn't this have the assets bundled together?

Comment (andrewoverton) [reply catalog/BUILD:134]: We spoke offline, the result being https://github.com/material-components/material-components-ios/issues/6416

**Pull request merged**